adops/setting-up-prebid-with-the-appnexus-ad-server.md

```diff
@@ -99,54 +102,65 @@ Follow the creative setup instructions in [Add Creatives](https://docs.xandr.com
 
-For each line item, follow the line item setup instructions in [Create a Line Item](https://docs.xandr.com/bundle/monetize_monetize-standard/page/topics/create-a-standard-line-item.html), with the following settings:
+For each line item, follow the line item setup setup instructions in [Create an Augmented Line Item (ALI)](https://docs.xandr.com/bundle/monetize_monetize-standard/page/topics/create-an-augmented-line-item-ali.html), with the following settings:
 
-* Set the **Revenue Type** to *CPM*.
+* Set the **Name** e.g., *€00.01 - Rubicon - Prebid Banner*.
 
-* Set the **Revenue Value** to one of the price bucket key-values from Step 1, e.g., \$0.2.
+* Set **Ad Type** Banner.
+
+* Set the **Revenue Type** to *CPM*.
 
-* Under **Associated Creatives**, choose to manage creatives at the line item level.
+* Set the **Revenue Value** to one of the price bucket key-values from Step 1, e.g., *0.01*.
 
-* Associate as many creative sizes with the line item as you need.  Set the **Creative Rotation** to *Even*.
+* Set the **Budget Setup** to *Unlimited Budget*.
 
-* In your line item's targeting settings, use the key-value that matches the line item's price bucket, e.g. (you set these up in Step 1).
+* Set the **End Date** sometime in the future e.g. *6/26/2033*.
 
-* Still in the targeting settings, target the custom category `prebid_enabled`. This will allow you to turn targeting on and off for a placement (or an entire placement group) by adding it to the custom category, which you'll do in one of the later steps.  This is useful for troubleshooting.
+* Within **Inventory & Brand Safety Targeting** and for **Supply Source** select *Managed Inventory*.
 
-For more information about targeting custom content categories, see [Content Category Targeting](https://docs.xandr.com/bundle/monetize_monetize-standard/page/topics/content-category-targeting.html).
+* Set the **Line Item Priority** for more information on line item priority see [Bidding Priority](https://docs.xandr.com/bundle/monetize_monetize-standard/page/topics/bidding-priority.html)
 
-## Step 4. Set up Campaigns
+* Optional: Still in the targeting settings, target the custom category `prebid_enabled`. This will allow you to turn targeting on and off for a placement (or an entire placement group) by adding it to the custom category, which you'll do in one of the later steps.  This is useful for troubleshooting.  For more information about targeting custom content categories, see [Content Category Targeting](https://docs.xandr.com/bundle/monetize_monetize-standard/page/topics/content-category-targeting.html).
 
-For each line item, create one campaign to associate with it.  The campaign should have an unlimited budget, start running right away, and run indefinitely.
+* In your line item's targeting settings, set the **Key/Value Targeting** to matche the line item's price bucket/revenue value.
```

Comment:
typo: 'matche'

Reply:
typo exists in the other doc as well. Convenient illustration of the hazard of duplicated text. :-)

Reply:
Corrected